.gitignore

```diff
@@ -9,3 +9,7 @@ styleguide/
 # Assets
 dist/
 
+# composer
+vendor
+composer.phar
+composer.lock
```

Comment:
This probably shouldn't be ignored, it enforces anyone using composer to have the same versions as you.